source/codegen/templates/service_helpers.mako

```diff
@@ -719,30 +713,41 @@ ${initialize_hardcoded_parameter(parameter)}
 
 
 ## Handles populating the response message after calling the driver API.
-<%def name="populate_response(function_data, output_parameters, indent_level=0, init_method=False)">\
+<%def name="populate_response(function_data, parameters, indent_level=0, init_method=False)">\
 <%
+  output_parameters = [p for p in parameters if common_helpers.is_output_parameter(p)]
   get_last_error_outputs = service_helpers.get_last_error_output_params(output_parameters)
   normal_outputs = [p for p in output_parameters if not p in get_last_error_outputs]
 %>\
 <%block filter="common_helpers.indent(indent_level)">\
+${populate_error_check(function_data, parameters, indent_level)}
```

Comment:
You shouldn't pass down the passed-in indent_level here. That would basically double it. I'd have to try it out (or dig around a while longer) to figure out the proper value, but my gut tells me you should leave it off in this case (so that it's 0).